patches/hmpo-form-wizard+15.0.0.patch

```diff
@@ -11,3 +11,16 @@ index 43130c1..77cb36a 100644
          this.router[method].apply(this.router, args);
      }
  
+diff --git a/node_modules/hmpo-form-wizard/lib/controller/mixins/next-step.js b/node_modules/hmpo-form-wizard/lib/controller/mixins/next-step.js
```

Comment:
Is this the same patch that Alex was commenting on the other day? Or is that on a different branch?

Reply:
yep, same one

Reply:
He was suggesting that this patchfile is badly formatted in some way. I can't see that conversation in this PR - was it resolved?

Reply:
The conversation was here https://mojdt.slack.com/archives/C06DX440XFY/p1747673490553029